test/fixtures/serializers/custom_item_links_serializer.rb

```diff
@@ -1,7 +1,9 @@
 class CustomItemLinksSerializer < CollectionJson::Serializer
-  links dashboard: {
-    href: "/my-dashboard",
-    anything: "at all",
-    whatever: "really"
-    }
+  items do
+    links dashboard: {
+      href: "/my-dashboard",
+      anything: "at all",
+      whatever: "really"
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.